src/Value.php

```diff
@@ -134,6 +134,7 @@ public function asNonEmptyStringOrNull(): ?string
     public function asNonEmptyStrings(): array
     {
         Assert::allStringNotEmpty($this->value);
+        Assert::isArray($this->value);
```

Comment:
Should probably be before the `Assert::all...`

Reply:
Changed